nav2_amcl/src/amcl_node.cpp

```diff
@@ -252,240 +193,10 @@ AmclNode::~AmclNode()
   // TODO(mhpanah): delete everything allocated in constructor
 }
 
-#if 0
```

Comment:
This has been removed in the header file but not from cpp file.

Reply:
I dropped that PR since the rename of amcl to amcl_localizer wasn't popular. Fine to make the changes here. The main things I was looking for was introducing a namespace for amcl, moving the header file to include, and eliminating dead code.